### PR TITLE
Overload series.rolling.apply()

### PR DESCRIPTION
**Pull request by densmirn**

### CODE REVIEW

File: sdc/datatypes/hpat_pandas_series_rolling_functions.py [hunk 1]
```diff
@@ -638,6 +685,35 @@ def culc_var(arr, ddof, minp):
...
+    'example_caption': 'Calculate the rolling apply.',
+    'example_result':
+    """
+        0    NaN
+        1    NaN
+        2    4.0
+        3    3.0
+        4    5.0
+        dtype: float64
+    """,
```
Comment (PokhodenkoSA) [inline sdc/datatypes/hpat_pandas_series_rolling_functions.py]: #455

File: sdc/datatypes/hpat_pandas_series_rolling_functions.py [hunk 2]
```diff
@@ -251,6 +257,47 @@ def impl(self):
     gen_hpat_pandas_series_rolling_impl(arr_sum, float64))
 
 
+@sdc_overload_method(SeriesRollingType, 'apply')
+def hpat_pandas_series_rolling_apply(self, func, raw=None):
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_series_rolling_functions.py]: Well. Actually `rolling.apply` allow to pass `args` and `kwargs` to function.It seems doesn't work right now in numba. @PokhodenkoSA  has details.

But please, at least specify this limitation in docs

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.core.window.Rolling.apply.html

**Comment by AlexanderKalistratov** [comment]
@densmirn sorry not saying it in the first comment. Could you please also add tests with args and skip it? So we wouldn't forget to fix it

**Comment by densmirn** [comment]
> @densmirn sorry not saying it in the first comment. Could you please also add tests with args and skip it? So we wouldn't forget to fix it

Done.

**Pull request merged**